### PR TITLE
fix(typography): use rem instead of px, add mobile size variant

### PR DESCRIPTION
**Pull request by hirsch88**

### CODE REVIEW

**Comment by incubator-tekton** [comment]
:rocket: Welcome to Baloise ChatOps based GitOps Bot :tada:
                                
    - If you want to enable preview environments for this PullRequest, comment with `/preview`. 
    - If you want to disable the preview environment for this PR, comment with `/undeploy`. 
    - If you need help or want to see the docs, comment with `/help`. 
                                
    Make sure to refresh this page after submitting a command.
    Preview environments can be redeployed automatically if you enable the webhook for updates to the source branch. :new:
    
    Have fun building awesome apps! :hammer:  :whale2:  :truck:  :ship:

**Review by nobilo — APPROVED** [review]

File: packages/components/src/components/bal-heading/bal-heading.scss [hunk 1]
```diff
@@ -26,7 +26,10 @@ h5,
 h6 {
   @include padding(0);
   font-family: $family-primary;
+  font-weight: 700;
```
Comment (yannickholzenkamp) [inline packages/components/src/components/bal-heading/bal-heading.scss]: no variable for font-weight?

File: packages/components/src/components/bal-heading/bal-heading.scss [hunk 2]
```diff
@@ -63,7 +66,7 @@ h6 {
 p,
 a {
   font-family: $family-primary;
-  line-height: 24px;
+  line-height: 1.5rem;
```
Comment (yannickholzenkamp) [inline packages/components/src/components/bal-heading/bal-heading.scss:68]: var?

**Review by yannickholzenkamp — APPROVED** [review]

File: packages/components/src/components/bal-heading/bal-heading.scss (hunk 2 above)
Comment (hirsch88) [reply packages/components/src/components/bal-heading/bal-heading.scss:68]: not for mobile

File: packages/components/src/components/bal-heading/bal-heading.scss (hunk 1 above)
Comment (hirsch88) [reply packages/components/src/components/bal-heading/bal-heading.scss]: $weight-bold, will add it. thanks

**Pull request merged**